data_steward/cdr_cleaner/cleaning_rules/deid/fitbit_pid_rid_map.py

```diff
@@ -22,9 +22,9 @@ def __init__(self,
                  project_id,
                  dataset_id,
                  sandbox_dataset_id,
-                 mapping_dataset_id,
-                 mapping_table_id,
-                 table_namer=None):
+                 table_namer=None,
+                 mapping_dataset_id=None,
+                 mapping_table_id=None):
```

Comment:
Before this change, I was getting the warning `FitbitPIDtoRID does not accept the ``table_namer`` property yet.`. And I noticed that `infer_rule` is assigning the fourth argument as `table_namer` [at this line](https://github.com/all-of-us/curation/blob/develop/data_steward/cdr_cleaner/clean_cdr_engine.py#L217). As a result of it, `infer_rule` was throwing `TypeError: __init__() got multiple values for argument 'mapping_dataset_id'` in the `try-except` clause.
So, I made this change to avoid the `TypeError` and that the `table_namer` works as expected. I am not 100% sure this is the right way to fix the table_namer issue. If you have any advice, I would really appreciate it.

Reply:
I assume you were seeing the error locally, since I can't find a stack trace on circleci.  How were you running this, because I haven't been able to reproduce the error with the original parameters?  I have been testing with logged in to the test environment with my account and the command line.
```bash
PYTHONPATH=./data_steward:$PYTHONPATH TEST_PATTERN=fitbit_pid_rid_map* UNIONED_DATASET_ID=<my_dataset_id> COMBINED_DATASET_ID=<other_dataset_id> ./tests/run_tests.sh integration
```

Reply:
@lrwb-aou I found the integration test last night in CircleCi is showing the `table_namer` warning: https://app.circleci.com/pipelines/github/all-of-us/curation/5809/workflows/aadd586e-32bc-4a38-ba89-30764960f07d/jobs/17534?invite=true#step-106-133

Reply:
[table_namer_fix.txt](https://github.com/all-of-us/curation/files/9143759/table_namer_fix.txt)
There is another part to the bug, but the fix isn't it `fitbit_pid_rid.py`.  It is in the clean engine itself.  This fixes the log message locally while still indicating the mapping parameters are required, not optional.